### PR TITLE
Added email to appsflyer when setting user

### PR DESCRIPTION
**Pull request by krakover**
Added setting of email (available in new appsflyer package)

### CODE REVIEW

**Comment by krakover** [comment]
@liorr please review

File: Providers/AppsFlyerProvider.m [hunk 1]
```diff
@@ -30,6 +30,9 @@ - (instancetype)initWithITunesAppID:(NSString *)iTunesAppID key:(NSString *)key
 
 - (void)identifyUserWithID:(NSString *)userID andEmailAddress:(NSString *)email {
     [AppsFlyerTracker sharedTracker].customerUserID = userID;
+    if (email) {
+        [[AppsFlyerTracker sharedTracker] setUserEmails:@[email] withCryptType:EmailCryptTypeSHA1];
```
Comment (liorr) [inline Providers/AppsFlyerProvider.m]: What exactly is the value of `withCryptType` used for?

Comment (krakover) [inline Providers/AppsFlyerProvider.m]: In case you want to encrypt it, and not show it in plain. I changed it to plain.

**Comment by krakover** [comment]
@liorr re-review?

**Comment by liorr** [comment]
@krakover what is the status with this PR?

**Comment by krakover** [comment]
@liorr I don't remember. We wanted to close it?

**Comment by liorr** [comment]
Yes

**Pull request closed without merging**